src/string/capitalize.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Converts the first character of string to upper case and the remaining to lower case.
+ *
+ * @param {string} str - The string to be converted to uppercase.
+ * @returns {string} - The capitalized string.
+ *
+ * @example
+ * const result = capitalize('fred') // returns 'Fred'
+ * const result2 = capitalize('FRED') // returns 'Fred'
+ */
+
+export const capitalize = (str: string): string => {
```

Comment:
The return type on this could be accurately be set to `Capitalize<string>`.

Reply:
Thank you for code review, updated type, can you please review again

Reply:
I'm just some random dude who is excited about this project and was poking around at the PRs, I have no authority to approve this code. I just thought it was simple improvement that could be made. (: